app/assets/javascripts/application.js

```diff
@@ -12,3 +12,16 @@
 //
 //= require rails-ujs
 //= require_tree .
+function ready(fn) {
+  if (document.readyState != 'loading'){
+    fn();
+  } else {
+    document.addEventListener('DOMContentLoaded', fn);
+  }
+}
+
+ready(function() {
+  document.getElementById("user_avatar").onchange = function () {
+    document.getElementById("uploadFile").value = this.value;
```

Comment:
Ha — this UI has a real retro feel. I like it! I guess you noticed the problem that there's no user feedback when they add an avatar, and that's why the text box is there?